include/zephyr/mgmt/hawkbit.h

```diff
@@ -37,6 +37,33 @@ enum hawkbit_response {
 	HAWKBIT_PROBE_IN_PROGRESS,
 };
 
+/**
+ * @brief hawkBit event type.
+ *
+ * @details These events are used to register the callback functions.
+ *
+ */
+enum hawkbit_event_type {
+	/** Event triggered when there was an error */
+	HAWKBIT_EVENT_ERROR,
+	/** Event triggered when a new update was installed */
+	HAWKBIT_EVENT_UPDATE_INSTALLED,
```

Comment:
I think that the naming of these _INSTALLED enums (response as well) are somewhat misleading, as we have two reboot and swap images before we can consider an update to be installed.

Suggestion: Rename to _DOWNLOADED.